users/permissions.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.permissions import BasePermission
+
+SAFE_METHODS = ['GET', 'HEAD', 'OPTIONS']
+
+
+class IsAuthenticatedAndOwner(BasePermission):
```

Comment:
Non authenticated users can not be owner of an object. Naming the permission class as `IsOwner` is enough.